apps/ffdweb-site/src/app/_styles/globals.css

```diff
@@ -269,4 +269,9 @@
   .meta {
     @apply text-neutral-50;
   }
+
+  /* SOCIAL */
+  .social-media-secondary {
```

Comment:
Shouldn't this be primary?

Reply:
Do we actually have a `secondary` or is it just `inherit`?

Reply:
Pushed a CR for this [>>](https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/1196/commits/105f05087ba3100552396c341d9d25e02e3205be) (Ignore the data changes - They are fixed in the Scout CR)